spring-integration-jdbc/src/main/resources/org/springframework/integration/jdbc/schema-postgresql.sql

```diff
@@ -58,3 +58,19 @@ CREATE TABLE INT_METADATA_STORE  (
 	REGION VARCHAR(100) NOT NULL,
 	constraint INT_METADATA_STORE_PK primary key (METADATA_KEY, REGION)
 );
+
+-- This is only needed if using PostgresChannelMessageSubscriber
+-- CREATE FUNCTION INT_CHANNEL_MESSAGE_NOTIFY_FCT()
+-- RETURNS TRIGGER AS
+-- $BODY$
+-- BEGIN
+--     EXECUTE 'NOTIFY int_channel_message_notify, ''' ||  QUOTE_LITERAL(NEW.REGION || NEW.GROUP_KEY) || '''';
```

Comment:
Why there is no some readable concatenator between region and group?
What if I have them like this: `myregion` and `mygroup` ?
So, with this I'll end up as `myregionmygroup`. something what is not end-user friendly 😢

Reply:
I thought that, since it's not exposed to the user, it does not matter much, but it's probably nice for debugging purposes. I added a space now.